### PR TITLE
Minor upgrade to /examples

### PR DESCRIPTION
**Pull request by m13m**
`to_char_list/1` has been deprecated

### CODE REVIEW

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: examples/tcp_connection/lib/tcp_connection.ex [hunk 1]
```diff
@@ -40,7 +40,7 @@ defmodule TCPConnection do
   end
 
   def connect(opts) do
-    host        = Keyword.fetch!(opts, :hostname) |> String.to_char_list()
+    host        = Keyword.fetch!(opts, :hostname) |> String.to_charlist()
```
Comment (fishcakez) [inline examples/tcp_connection/lib/tcp_connection.ex]: Should bump the elixir requirement in the mix.exs as this is not in 1.0.

Comment (m13m) [reply examples/tcp_connection/lib/tcp_connection.ex]: @fishcakez  Should I bump it to elixir 1.6.2 on both examples

**Comment by josevalim** [comment]
~> 1.4 should be enough.
-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

**Pull request merged**